models/junos/security_policies/junos_security_policies.yml

```diff
@@ -0,0 +1,541 @@
+---
+module: junos_security_policies
+version_added: 212
+short_description: Create and manage security policies on Juniper JUNOS devices
+description: This module provides declarative creation and management of security policies on Juniper JUNOS devices
+author: Pranav Bhatt (pbhatt@redhatcom)
+requirements:
+  - ncclient (>=v064)
+notes:
+  - This module requires the netconf system service be enabled on the device being managed
+  - This module works with connection C(netconf) See L(the Junos OS Platform Options,https://docsansiblecom/ansible/latest/network/user_guide/platform_junoshtml)
+  - Tested against JunOS v184R1
+options:
+  config:
+    description: A dictionary of security policies
+    type: list
+    elements: dict
+    suboptions:
+      policy_type:
```

Comment:
type is missing. type: str

Reply:
I'm taking advantage of the default type for `choices` being `str` here. If it is frowned up though, I can add that explicitly